### DISCUSSION TITLE
how accurate are the sky maps for 10-15k years AD or BC?

### DISCUSSION THREAD

**Discussion by hvnlystarz** [Stellarium/stellarium#2326] [Q&A]
i am tracking some constellations through the ages- and i am not sure if what i am seeing is accurate or a guesstimate from this software or perhaps even just random?  it allows me to go up to 50000 years AD or 45000 years BC (maybe more or less i have yet to track it) but want to know how accurate the info is i am getting - and if i can rely on it for my research?  thank you!

**Comment by gzotti** [comment]
Appendix F in the User Guide tells you what the authors of the program know about its accuracy. Please find other software and compare, then tell us what is better and why.

**Reply by hvnlystarz** [reply]
thank you i will read that!  i don't know other software to use?  i want to get my hands on NASA software but don't have the means lol.  most software for astrology charts only goes back 3200 BC and then up to 3500 AD or so.  do you recommend i check anything else out?  i am all ears.  thanks!

**Reply by Atque** [reply]
I would recommend Cartes du Ciel for this purpose. Its default star catalog has fully modelled 3D proper motion, which even takes into account variations in brightness through epochs. If you want planetary accuracy, you'll need the DE431/DE441 ephemeris, which is valid for 13,000 BC to 17,000 AD. Outside that range, I don't know if there is any ephemeris available at the moment.

If you simply want a website, you might want to try this one:
http://orbitsimulator.com/gravitySimulatorCloud/properMotionHome.html

But keep in mind that its precession model is inferior compared to Stellarium and CdC's. It also completely lacks planetary movements.

**Reply by hvnlystarz** [reply]
thank you atque- i will look into both

do you by chance know what software nasa uses and if it is accessible to lay people in some version or format?

**Reply by gzotti** [reply]
I doubt NASA is interested in planet positions beyond NASA/JPL's DE441.

**Reply by hvnlystarz** [reply]
i am not interested in planet positions- but star positions/constellations and precession.  i think NASA is interested in that

**Reply by Atque** [reply]
Then you should look into the Gaia project. However, the full data release is scheduled in 2025.

**Reply by hvnlystarz** [reply]
thank you atque- i have not heard of this but looking into it now.
